flair/trainers/language_model_trainer.py

```diff
@@ -418,14 +425,15 @@ def evaluate(self, data_source, eval_batch_size, sequence_length):
             total_loss = 0
             ntokens = len(self.corpus.dictionary)
 
-            hidden = self.model.init_hidden(eval_batch_size)
+            hidden = self.model.module.init_hidden(eval_batch_size)
 
             for i in range(0, data_source.size(0) - 1, sequence_length):
                 data, targets = self._get_batch(data_source, i, sequence_length)
                 prediction, rnn_output, hidden = self.model.forward(data, hidden)
                 output_flat = prediction.view(-1, ntokens)
                 total_loss += len(data) * self.loss_function(output_flat, targets).data
```

Comment:
If using multiple GPUs, we need to align tensors to be on the same device, I received an issue here.

Reply:
`flair.device = self.device` will move the tensor to appropriate device with multiple devices. I have added the fix. :)